rpcs3/rpcs3qt/main_window.h

```diff
@@ -84,6 +84,9 @@ class main_window : public QMainWindow
 	bool Init();
 	~main_window();
 	QIcon GetAppIcon();
+	bool OnMissingFw();
+
+	static bool HandlePupInstallation(main_window* mw, QString file_path = "");
```

Comment:
all of the main window checks are useless and this shouldn't be a static function.

Reply:
Removed.